kura/org.eclipse.kura.camel.cloud.factory/META-INF/MANIFEST.MF

```diff
@@ -19,6 +19,7 @@ Import-Package: org.apache.camel;version="[2.17.2,3.0.0)",
  org.eclipse.kura.cloud.factory;version="[1.1,1.2)",
  org.eclipse.kura.configuration;version="[1.1,1.2)",
  org.eclipse.kura.configuration.metatype;version="[1.0,2.0)",
+ org.eclipse.kura.core.configuration;version="1.0.2",
```

Comment:
@ctron why we are not using version range here for backward compatibility reasons?
Thanks